sys/include/net/gnrc/rpl.h

```diff
@@ -279,22 +284,25 @@ static inline bool GNRC_RPL_COUNTER_GREATER_THAN(uint8_t A, uint8_t B)
  * default lifetime will be multiplied by the lifetime unit to obtain the resulting lifetime
  * @{
  */
-#define GNRC_RPL_DEFAULT_LIFETIME (60)
-#define GNRC_RPL_LIFETIME_UNIT (2)
+#ifndef GNRC_RPL_DEFAULT_LIFETIME
+#define GNRC_RPL_DEFAULT_LIFETIME (5)
```

Comment:
Btw. Contiki uses a default lifetime of 30*60s = 30min, just to provide some other perspective on this. And IoT is about saving energy, messages and so on, so we don't want to trigger lots of control traffic by default.